tests/test_host.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+
+from illuminatio.host import (
+    ClusterHost,
+    ConcreteClusterHost,
+    ExternalHost,
+    LocalHost,
+    GenericClusterHost,
+    Host,
+)
+
+
+@pytest.mark.parametrize(
+    "identifier,expected_host",
+    [
+        pytest.param("localhost", LocalHost(), id="LocalHost"),
+        pytest.param(
+            "127.0.0.1", ExternalHost("127.0.0.1"), id="ExternalHost with localhost IP"
+        ),
+        pytest.param(
+            "123.123.123.123",
+            ExternalHost("123.123.123.123"),
+            id="ExternalHost with long IP",
```

Comment:
Also the id is wrong? It's just a normal IP not a "long" IP.

Reply:
Compared to the IP address in the test afterwards its long. Though the usage of the ipaddress library probably makes using a second test obsolete

Reply:
I know what you mean but I wouldn't call this a 'long' and a 'short' IP and I also thing we can drop one of the test cases since thee ipaddress library should handle this.